.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -47,6 +47,10 @@ parameters:
     type: string
     default: ""
 
+  - name: PTF_IMAGE_TAG
+    type: string
+    default: ""
```

Comment:
I think for community builds the default image tag must be "latest". When this is active and empty tag is passed down to the PR tests the PTF image pull would fail. Unless it has been set to "latest" elsewhere in elastictest.